core/sql/exp/ExpHbaseInterface.cpp

```diff
@@ -271,7 +271,12 @@ Lng32 ExpHbaseInterface::coProcAggr(
 
 char * getHbaseErrStr(Lng32 errEnum)
 {
-  return (char*)hbaseErrorEnumStr[errEnum - (Lng32)HBASE_MIN_ERROR_NUM];
+  Lng32 lv_errEnum;
+  if (errEnum < HBASE_MIN_ERROR_NUM || errEnum >= HBASE_MAX_ERROR_NUM)
+     lv_errEnum = HBASE_GENERIC_ERROR; 
```

Comment:
It is good that we are adding this to catch unanticipated errors. Did testing reveal any errors that are not on the list above. If yes, do you think we can add to the list of errors in ExpHbaseDefs.h.